### PR TITLE
Adjusts some of the contraband weights for Contraband Crates.

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

This makes getting Contraband Plus (the better stuff with more interesting effects), slightly higher in probability out of contraband crates.

It also removes the completely redundant narcotics roll, and the truly awful permabrig contraband roll, from Contraband Plus.

Some of the more interesting narcotics are now directly in Contraband Plus.

## Why It's Good For The Game

The weights were a little uninteresting. You rarely if ever saw the effects of the contraband crate change with the current values. It was kind of lame to see a shiv in a contraband crate knowing that it was a wasted Contraband Plus roll. This hopefully stops that from happening too often.

## Changelog

:cl:
balance: Adjusts the values and contents of the Contraband Crate's item loot table.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/spawners/random/contraband.dm [hunk 1]
```diff
@@ -21,22 +21,20 @@
...
-		/obj/effect/spawner/random/contraband/prison = 40,
 		/obj/item/clothing/under/syndicate = 20,
 		/obj/item/reagent_containers/cup/bottle/thermite = 20,
-		/obj/item/reagent_containers/pill/maintenance = 10,
 		/obj/item/restraints/legcuffs/beartrap = 10,
-		/obj/effect/spawner/random/contraband/narcotics = 10,
-		/obj/item/seeds/kronkus = 5,
-		/obj/item/seeds/odious_puffball = 5,
+		/obj/item/food/drug/saturnx = 5,
+		/obj/item/reagent_containers/cup/blastoff_ampoule =5,
```
Comment (san7890) [inline code/game/objects/effects/spawners/random/contraband.dm:36]:
```suggestion
		/obj/item/reagent_containers/cup/blastoff_ampoule = 5,
```

**Review by san7890 — CHANGES_REQUESTED** [review]
let's be consistent

**Comment by dalmationer** [comment]
I believe this also cuts out the syndicate mask, right? Any chance that could be left in, as it lets you complete the look if you get the turtleneck too!

**Comment by necromanceranne** [comment]
The mask is in normal contraband.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**